var/spack/repos/builtin/packages/llvm/package.py

```diff
@@ -52,7 +52,7 @@ class Llvm(Package):
     depends_on('cmake @2.8.12.2:')
 
     # Universal dependency
-    depends_on('python@2.7:')
+    depends_on('python@2.7:2.8')  # Seems not to support python 3.X.Y
```

Comment:
@mathstuf Are you sure this will include anything that matches 2.7.X? I just tried and got:
```
==> Warning: There is no checksum on file to fetch python@2.7 safely.
  Fetch anyway? [y/N]
```
Another issue is, I think, that somehow the `preferred` keyword got lost in the versioning. If I try : 
```
spack install python
```
spack installs `3.5.1`, even if in the `python` package I see : 
```
version('2.7.11', '6b6076ec9e93f05dd63e47eb9c15728b', preferred=True)
```